packages/grid_client/scripts/applications/nostor.ts

```diff
@@ -0,0 +1,135 @@
+import { GatewayNameModel, GridClient, MachinesModel } from "../../src";
+import { config, getClient } from "../client_loader";
+import { log, pingNodes } from "../utils";
+
+async function deploy(client: GridClient, vms: MachinesModel, subdomain: string, gatewayNode: any) {
```

Comment:
```suggestion
async function deploy(client: GridClient, vms: MachinesModel, subdomain: string, gatewayNode: NodeInfo) {
```
please update this type its better to avoid any

Reply:
The gatewayNode's type is still any, please change it

Reply:
please make sure to change the type here